src/alert_rules/prometheus/kafka_metrics.rules

```diff
@@ -69,8 +69,8 @@ groups:
   # Controller and Partitions
   # =========================
   - alert: Active Controllers
-    expr: sum(kafka_controller_kafkacontroller_activecontrollercount{juju_charm!=".*"}) != 1
-    for: 1m
+    expr: sum(kafka_controller_kafkacontroller_activecontrollercount{juju_charm!=".*"}) == 0
+    for: 0m
```

Comment:
**question** are we sure that this does not create a lot of noise during elections? Shall we maybe increase the `for: ` to `1m` or even just 10 or 30 seconds?